### PR TITLE
[Integration test][ome_application_network_proxy][Module] - All features

### PR DESCRIPTION
**Pull request by Saksham-Nautiyal**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: tests/integration/targets/ome_application_network_proxy/tests/set.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        enable_authentication: false
+      register: reset_network_proxy_authentication
+
+    - name: Reset proxy configuration.
+      ome_application_network_proxy:
+        enable_proxy: false
+      register: reset_network_proxy_config
+
+    - name: Verify the status of tasks for enable_ntp and set addresses
+      assert:
```
Comment (rajshekarp87) [inline tests/integration/targets/ome_application_network_proxy/tests/set.yaml]: Move all the assertion under respective task. Make this changes at all applicable places

File: tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+      register: configure_network_proxy
+
+    - name: Verify Update ignore_certificate_validation (normal mode)
+      ansible.builtin.assert:
+        that:
+          - configure_network_proxy.changed
+          - configure_network_proxy.msg == "Successfully updated network proxy configuration."
+
+    - name: Get application network time details by uri.
+      uri:
```
Comment (ShivamSh3) [inline tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml]: use FQDN :
ansible.builtin.uri

File: tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        status_code: 200
+      register: uri_proxy
+
+    - name: Verify - Get application network time details by uri
+      ansible.builtin.assert:
+        that:
+          - configure_network_proxy.changed
+          - configure_network_proxy.proxy_configuration.SslCheckDisabled ==
+           uri_proxy.json.SslCheckDisabled
+          - uri_proxy.json.IpAddress ==
```
Comment (rajshekarp87) [inline tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml]:
```suggestion
          - configure_network_proxy.proxy_configuration.IpAddress ==
```

File: tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+           configure_network_proxy.proxy_configuration.IpAddress
```
Comment (rajshekarp87) [inline tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml]:
```suggestion
           uri_proxy.json.IpAddress
```

Comment (rajshekarp87) [reply tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml]: Incorporate this comment at all applicable places

File: tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+  when: hostname is not defined or username is not defined or password
+         is not defined
+
+- block:
+    # PRE REQ - CONFIGURE NETWORK PROXY
+    - name: Reset proxy configuration.
+      dellemc.openmanage.ome_application_network_proxy:
+        enable_proxy: false
+
+    - name: Update ignore_certificate_validation. (check mode - changes expected)
```
Comment (rajshekarp87) [inline tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml]: Limit the line length to 80 character. Make this changes at all applicable places

File: tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml (hunk 2 above)
Comment (Saksham-Nautiyal) [reply tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml]: donedone

File: tests/integration/targets/ome_application_network_proxy/tests/set.yaml (hunk 1 above)
Comment (Saksham-Nautiyal) [reply tests/integration/targets/ome_application_network_proxy/tests/set.yaml]: updated

File: tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml (hunk 3 above)
Comment (Saksham-Nautiyal) [reply tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml]: updated

File: tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml (hunk 5 above)
Comment (Saksham-Nautiyal) [reply tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml]: updated

File: tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml (hunk 4 above)
Comment (Saksham-Nautiyal) [reply tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml]: updated

**Comment by ShivamSh3** [comment]
image

**Comment by Saksham-Nautiyal** [comment]
For ipv4

image

**Comment by Saksham-Nautiyal** [comment]
For ipv6

image

**Comment by singh-lovepreet1** [comment]
Resolve Github actions if applicable

**Review by sachin-apa — DISMISSED** [review]

**Review by ShivamSh3 — DISMISSED** [review]

**Review by sachin-apa — DISMISSED** [review]

**Review by singh-lovepreet1 — APPROVED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Review by ShivamSh3 — APPROVED** [review]

**Pull request merged**